### PR TITLE
Psg 4892 Use repo templates in passage-react-native

### PR DESCRIPTION
**Pull request by jenn-macfarlane**
## What's New?

Uses new repo templates/repo guidelines - removes COC + Contributing files, updates License file, package.json and Readme.

## Screenshots (if appropriate):

(Passage logo will respect theme, this is from vscode preview):
CleanShot 2024-10-18 at 12 42 27@2x

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have manually tested my code thoroughly
- [ ] I have added/updated inline documentation for public facing interfaces if relevant
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing integration and unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Additional context

Add any other context or screenshots about the pull request here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,78 @@
...
+
+### In passage-react-native
+
+Use passage-react-native to implement Passkey Complete into your React Native app to authenticate requests and manage users.
+
+| Product | Compatible |
+| --- | --- |
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex** | ✖️ For Passkey Flex, check out [passage-flex-react-native](https://github.com/passageidentity/passage-flex-react-native)
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✅
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready** | ✖️ For Passkey Ready, check out [Authentikit](https://www.npmjs.com/package/@passageidentity/authentikit)
```
Comment (jenn-macfarlane) [inline README.md:27]: Still unclear on which Authentikit packages we should point to - I imagine for React Native pointing to the web version of Authentikit makes sense, but for native mobile READMEs we should be using Android/iOS Authentikit packages, right?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2023 Ricky Padilla
```
Comment (rickycpadilla) [inline LICENSE:3]: lol

File: README.md (hunk 1 above)
Comment (rickycpadilla) [reply README.md:27]: We should point to native iOS and native android for Authentikit :)

File: LICENSE (hunk 2 above)
Comment (jenn-macfarlane) [reply LICENSE:3]: 🙈 your legacy is everywhere don't worry!! Hahah

File: README.md (hunk 1 above)
Comment (jenn-macfarlane) [reply README.md:27]: Sounds good, same as Flutter readme. Thanks!!

**Review by rickycpadilla — APPROVED** [review]
Thanks Jenn!

**Comment by jenn-macfarlane** [comment]
Waiting to merge until the [Android](https://passage-identity.atlassian.net/browse/PSG-4895) ticket is done. Reason being this way I will have a README for Android/authentikit to link to in the products comparison table instead of just the folder in github

**Pull request merged**